package.json

```
@@ -15,31 +15,36 @@
     "test-coverage": "nyc npm run test",
     "test-coverage-ci": "nyc --reporter=lcovonly --reporter=text npm run test",
     "prepare": "node ./scripts/prepare.js",
-    "lint": "eslint --fix . --ext .js,.jsx",
+    "lint": "eslint \"src/**/*.{js,jsx,ts,tsx,json}\" \"test/**/*.{js,jsx,ts,tsx,json}\"",
+    "lint:fix": "eslint --fix \"src/**/*.{js,jsx,ts,tsx,json}\" \"test/**/*.{js,jsx,ts,tsx,json}\"",
+    "format": "prettier --write src/**/*.{js,jsx,ts,tsx,css,md,json,scss} test/**/*.{js,jsx,ts,tsx,json} --config ./.prettierrc",
     "gen-schema-doc": "node ./scripts/doc-schema.js"
   },
-  "bin": "./index.js",
+  "bin": {
+    "git-proxy": "./index.js",
+    "fullstack": "concurrently 'npm run server' 'npm run client'"
```

A better name here is needed. Testing this, it creates an executable with the literal name `fullstack`. It's not clear that this is from `@finos/git-proxy` and may cause more confusion. 

```bash
$ npm link
$ which fullstack
/home/tom/.asdf/shims/fullstack
```

Also, the executable cannot resolve `concurrently` for some reason and I wonder if this is just better ran from source until the whole application's entrypoint can be rewritten to allow these services to be selectively turned off. I do not believe `npm run start` is designed to be ran outside of the repo source since it relies on npm for setting the PATH to `./node_modules/.bin`.

This isn't really feasible for 1.x IMO.

I'm suggesting a better name at least but I suspect this executable will be challenging to get right with the current startup commands (running via `vite` and the current index.js entrypoint script).
```suggestion
     "git-proxy-all": "concurrently 'npm run server' 'npm run client'"
```

---

I'd like to propose using `gp` to shorten `git-proxy` , and therefore having `gp-all` instead of `git-proxy-all`, which coulld be longer to type. WDYT?

---

What about `gp-shallow` and `gp-all`?